.circleci/config.yml

```diff
@@ -102,10 +118,39 @@ jobs:
       - run:
           name: run add-chain module unit tests
           command: just test-add-chain
+      - notify-failures-on-main
+  golang-validate-modified:
+    shell: /bin/bash -eo pipefail
+    executor:
+      name: go/default  # is based on cimg/go
+      tag: '1.21'
+    steps:
+      - checkout
+      - install-just
+      - run:
+          name: Install gotestsum
+          command: go install gotest.tools/gotestsum@latest
```

Comment:
This can probably be removed #456

Reply:
cleaned up all invocations of `Install gotestsum`